test/smoketests/spectests/failures/failures.wast

```diff
@@ -40,9 +40,15 @@
 ;; invalid result
 (module
     (func (export "foo.i32") (result i32) (i32.const 1))
+    (func (export "foo.i64") (result i64) (i64.const 1))
+    (func (export "foo.f32") (result f32) (f32.const 1))
+    (func (export "foo.f64") (result f64) (f64.const 1))
```

Comment:
I think it would be better to use actual floats for these, to be sure. Such as 1.234.

Reply:
Changed